ansible/MANUAL_STEPS.md

```diff
@@ -108,8 +108,13 @@ Xcode Command-line tools are not enough to perform a full notarization cycle, fu
 As root:
 
 * Download Xcode: https://developer.apple.com/download/more/ - find non-beta version, open Developer Tools in browser, Networking tab, start download (then cancel), in Networking tab "Copy as cURL" (available in Chrome & FF)
+  * On OSX 11 we currently install 13.2.1
```

Comment:
We do it on OSX 10.15 as well.

Reply:
Was it reset too? It's annoying as I spent hours installing the same version on all of them

Reply:
I'm not aware of the nearform machines being reconfigured/reset. 

It does look like the orka 10.15 release machine has been reset though and needs a rebuild/restore (currently offline in ci-release) 😞.

Reply:
I was wrong. I only updated macOS 11 machines: https://github.com/nodejs/build/issues/3311#issuecomment-1529709626